RevenueCatUI/Templates/V2/Components/Stack/StackComponentViewModel.swift

```diff
@@ -45,6 +44,16 @@ class StackComponentViewModel {
         self.presentedOverrides = try self.component.overrides?.toPresentedOverrides { $0 }
     }
 
+    func copy(withViewModels newViewModels: [PaywallComponentViewModel]) throws -> StackComponentViewModel {
+        return try StackComponentViewModel(
+            component: self.component,
+            viewModels: newViewModels,
+            badgeViewModels: self.badgeViewModels,
+            shouldApplySafeAreaInset: self.shouldApplySafeAreaInset,
+            uiConfigProvider: self.uiConfigProvider
+        )
+    }
```

Comment:
This is the key new method that allows to clone a stack view model changing its child view models.